Sources/Apodini/Request/Dynamics.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Dynamics.swift
+//  
+//
+//  Created by Max Obermeier on 10.12.20.
+//
+
+import Foundation
+
+public struct Dynamics {
+
+    internal var elements: [String: Any]
```

Comment:
I would add a generic parameter here and constrain it to dynamic property.
Since the dynamic property protocol has no associated types worst case the developer can use `@Dynamics var dynamics: [String : DynamicProperty]`
 
Will make it harder to make a mistake and add something that can't be injected in here:

```swift
public struct Dynamics<Element : DynamicProperty> : Collection {
    var elements: [String : Element]
}
```

Reply:
I also thought about adding a generic, but constraining it to `DynamicProperty` makes no sense to me as e.g. `@Parameter` is no `DynamicProperty`.